pages/common/uniq.md

```diff
@@ -1,6 +1,7 @@
 # uniq
 
-> Report or omit repeated lines.
+> Output the unique lines from the given input or file.
+> Compare the adjacent files in input (file or standard input) and outputs unique lines.
 
```

Comment:
Does it really makes sense to have such long description?
I like the original description or just a first line of your description. I think that 2nd line of your description does not add too much and hence I'd rather remove it.